docs/docs/components/__tests__/expression-metadata.test.js

```diff
@@ -0,0 +1,24 @@
+import { types } from '../expression-metadata';
+import ref from '../../../../tsc/src/reference/latest';
+
+// help us catch that every type in expression-metadata.js exists in the style-spec
+// otherwise the site will not build
+describe('expression-metadata.js type exists in maplibre-gl-js-style-spec', () => {
+    for (const name in types) {
+        it(name, () => {
+            expect(ref['expression_name'].values[name]).not.toBeUndefined();
+        });
+    }
+});
+
+// help us catch expressions we need to add to expression-metadata.js during version bumps
```

Comment:
I'm sure there are better ways to do it instead of catching it in a test - i.e. better automation...